packages/algorithm/package.json

```diff
@@ -56,17 +57,17 @@
     "karma-mocha": "^2.0.1",
     "karma-mocha-reporter": "^2.2.5",
     "mocha": "^9.0.3",
+    "postcss": "^8.4.14",
     "rimraf": "^3.0.2",
-    "rollup": "^2.56.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-postcss": "^4.0.0",
+    "rollup": "^2.77.2",
+    "rollup-plugin-postcss": "^4.0.2",
     "rollup-plugin-sourcemaps": "^0.6.3",
     "terser": "^5.7.1",
-    "tslib": "^2.3.0",
+    "tslib": "^2.4.0",
     "typedoc": "~0.23.9",
     "typescript": "~4.7.3",
-    "webpack": "^4.41.3",
-    "webpack-cli": "^3.3.10"
+    "webpack": "^5.74.0",
+    "webpack-cli": "^4.10.0"
```

Comment:
It is probably out of scope for this PR. But it will be nice to drop webpack because it is inconsistent to distribute lumino using rollup but testing it using webpack.